nbconvert/exporters/base.py

```diff
@@ -136,7 +134,7 @@ def get_export_names(config=get_config()):  # noqa
     Exporters can be found in external packages by registering
     them as an nbconvert.exporter entrypoint.
     """
-    exporters = sorted(entrypoints.get_group_named("nbconvert.exporters"))
+    exporters = (e.name for e in entry_points(group="nbconvert.exporters"))
```

Comment:
this is a generator: should probably still be sorted, as well...

Reply:
indeed, fixed